styles/index.css

```diff
@@ -0,0 +1,114 @@
+:root {
+    --main-bg-color: rgb(16, 3, 46);
+    --starwars-yellow: rgb(88%, 69%, 0%);
+    --logo-width: 120px ;
+}
+
+/* BACKGROUND */
+body {
+    background-color: var(--main-bg-color);
+    margin: 0;
+}
+
+
+.main-bg-image {
+    width: 100%;
+    height: 170%;
+    position: absolute;
+    background-position: center;
+    background-image: url("/images/background_stars.webp");
+    background-size: cover;
+    background-repeat: no-repeat;
+    backdrop-filter: var(--main-bg-color);
+    left: 0;
+}
+
+#main-grid-container {
+    max-width: 1440px;
+    display: grid;
+    align-items: center;
+    justify-items: center;
+    grid-template-columns: 1fr 5fr;
+    grid-template-rows: auto 1fr;
+    grid-template-areas:
+        "logo     navbar "
+        "content content ";
+}
+
+.sw-logo {
+    grid-area: logo;
+    align-self: start;
+    background-size: 120px;
+    width: var(--logo-width);
+    height: 80px;
+    background-image: url("/images/starwars_logo.png");
+    background-repeat: no-repeat;
+    margin: 4em 2em 2em 2em;
+}
+
+/* QUIZ MENU */
+.quiz-main-menu {
+    margin-top: 1.5em;
+    box-sizing: border-box;
+    grid-area: navbar;
+    display: flex;
+    margin-right: 1em;
+    border: solid var(--starwars-yellow) 0.5rem;
+    border-radius: 3em;
+    background: rgba(0, 0, 0, 0.40);
+    box-shadow: 1px 1px 120px var(--starwars-yellow);
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+}
+
+
+/* QUIZ MENU CONTENT */
+button {
+    cursor: pointer;
+}
+
+.btn {
```

Comment:
You could make the class name even more specific to differentiate these buttons more from the rest of the buttons in the project, using a name like `btn--navbar` or `btn-header`.